components/mpas-seaice/src/shared/mpas_seaice_forcing.F

```diff
@@ -1181,6 +1186,8 @@ subroutine init_oceanic_forcing(domain)
     select case (trim(config_forcing_sst_type))
     case ("ncar")
        call init_oceanic_forcing_ncar(domain)
+    case ("ncar_free_surface")
+       call init_oceanic_forcing_ncar_free_surface(domain)
```

Comment:
I think the nomenclature here is incorrect.  POP is a free-surface ocean model, although it is constant volume.  So the original ncar forcing data is usable for a "free surface" ocean model.

Reply:
Good point, I have renamed the option to som_varying_thickness